deployment/network-operator/templates/upgrade-crd.yaml

```diff
@@ -1,11 +1,54 @@
 {{- if .Values.upgradeCRDs }}
+---
+apiVersion: v1
```

Comment:
we were missing this from original PR ?

Reply:
No, we were using main service-account of network-operator. The problem is that this account is not exist during pre-install hook

Reply:
ack, 

should we suffix them with <full-name>__-hooks__-sa/role/binding ?

Reply:
This doesn't really matter, because this objects will be immediately removed after completion of the hook. But I renamed them for clarity